src/components/create-deck/create-deck.js

```diff
@@ -0,0 +1,121 @@
+import EditPanel from '../edit-panel/edit-panel';
+import { Component } from 'react';
+
+import './create-deck.css';
+
+const store = require('store');
+
+class CreateDeck extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            eng: '',
+            rus: '',
+            deck: [],
+            deckTitle: ''
+        };
+        this.maxId = 1;
+    }
+
+    onValueChange = (e) => {
+        this.setState({
```

Comment:
you re-render the whole page after changing any symbol, it's very expensive
try to use hooks and get rid of it